src/modules/CRUD/components/Details/PropertyItem/Type/Type.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ArrayType from './ArrayType';
+import BooleanType from './BooleanType';
+import FileType from './FileType';
+import ObjectType from './ObjectType';
+import StringType from './StringType';
+
+export const TYPES = {
+  array: ArrayType,
+  boolean: BooleanType,
+  file: FileType,
+  image: FileType,
+  number: StringType,
+  object: ObjectType,
+  string: StringType,
+};
+
+export const getComponent = type => {
+  if (type in TYPES) {
+    return TYPES[type];
+  }
+
+  // eslint-disable-next-line no-console
+  console.warn(`type ${type} is invalid`);
+  return () => null;
+};
+
+export const getRightType = (schemaType, type) => {
+  if (['file', 'image'].includes(type)) {
+    return type;
+  }
+  return schemaType || type;
+};
+
+const Type = props => {
+  const {
+    schema: {
+      type: schemaType,
+    },
+    type,
+  } = props;
```

Comment:
Same as https://github.com/Terralego/terra-admin/pull/201/files#r378900295

Reply:
Agreed for the previous review, but in this case all `props` are forwarded to `Component` below